tvb_contrib/tvb/contrib/cosimulation/cosimulator.py

```diff
@@ -91,14 +112,8 @@ def _configure_cosimulation(self):
            - configure the cosimulation monitor
            - zero connectivity weights to/from nodes modelled exclusively by the other cosimulator
            """
-        # the synchronization time should be at least equal to integrator.dt:
-        self.synchronization_time = numpy.maximum(self.synchronization_time, self.integrator.dt)
-        # Compute the number of synchronization time steps:
-        self.synchronization_n_step = iround(self.synchronization_time / self.integrator.dt)
-        # Check if the synchronization time is smaller than the delay of the connectivity
-        # the condition is probably not correct. It will change with usage.
-        if self.synchronization_n_step > numpy.min(self.connectivity.idelays[numpy.nonzero(self.connectivity.idelays)]):
-            raise ValueError('the synchronization time is too long')
+        # Configure the synchronizatin time and number of steps
+        self._configure_synchronization_time()
```

Comment:
this is good.  reducing the length of complicated configuration functions by piecing them into smaller, well named methods bits is good.